test/binaries/Makefile

```diff
@@ -6,5 +6,8 @@ all: $(OBJS)
 %.out: %.c
 	gcc -o $@ $<
 
-clean:
+clean-linux:
```

Comment:
Can we make it so "make clean" just does the right thing based on platform?  It doesn't matter for the automated tests, but it's such a convention when using make that I would expect people to run it manually when they're experimenting.  Make is pretty powerful so I think it can probably be done?

Reply:
Sorry I'm not sure I understand you correctly, right now user doesn't need to run commands like `make clean-linux`, the test script will automatically run the clean command based on the platform the user is using, `clean-linux` and `clean-windows` are two flags for python to identify. Or I could also integrate them together like
```
clean:
    rm *.out
    erase *.out
```

Reply:
Users don't *need* to run the commands, but I'm saying that they likely will when doing their own testing because that's just what developers are like.  But you know what, this doesn't matter enough to hold up the patch, so we can always file a bug and fix it later.